src/components/MediaList/MediaList.tsx

```diff
@@ -160,21 +165,33 @@ const isLink = (url: string) => url.startsWith("http") || url.startsWith("https"
 interface HandleLinkProps {
   url: string;
   children: ReactNode;
+  type: ItemTypeWithDescription["metaType"];
 }
 
-export const HandleLink = ({ url, children }: HandleLinkProps) => {
+const licenceTag = (type: ItemTypeWithDescription["metaType"], url: boolean): string | undefined =>
+  ({
+    title: "dct:title",
+    author: "cc:attributionName",
+    copyrightHolder: "cc:copyrightHolder",
+    contributor: "cc:contributor",
+    other: url ? "cc:attributionURL" : undefined,
+    //@ts-ignore
```

Comment:
Hvorfor trenger vi en ts-ignore her?

Reply:
Pga typen ble laget ved å bruke `Omit` og det liker ikke typescript akkurat her. Jeg kunne egentlig bytte til `Exclude` for akkurat i denne casen furgerer de likt